docs/architecture/klickerKnight.md

```diff
@@ -0,0 +1,20 @@
+# Klicker Knight
+
+[klickerKnight.ts](../src/klickerKnight.ts) represents the entry point to the game.
+It abstracts the string input and output from the [GameUtil](./gameUtil.md).
+
+## npm run klicker-knight
+
+```mermaid
+sequenceDiagram
+  autonumber
+  actor D as Developer
```

Comment:
Nitpicking: Should `Developer` be `Player` here? (We have been using the "player" term in the room docs.

Reply:
Updated